integration-cli/requirements.go

```diff
@@ -139,11 +139,11 @@ var (
 	}
 	NotGCCGO = testRequirement{
 		func() bool {
-			out, err := exec.Command("go", "version").Output()
-			if err != nil && strings.Contains(string(out), "gccgo") {
-				return true
+			out, _ := exec.Command("go", "version").Output()
```

Comment:
I'd rather leave the err check and return `false` if `err != nil` here, the continue with your logic